src/App.tsx

```diff
@@ -51,9 +77,15 @@ const DashboardSidebar = () => {
         {astarte.token?.can('realmManagement', 'GET', '/triggers') && (
           <Sidebar.Item label="Triggers" link="/triggers" icon="triggers" />
         )}
-        {astarte.token?.can('realmManagement', 'GET', '/policies') && (
-          <Sidebar.Item label="Delivery Policies" link="/trigger-delivery-policies" icon="policy" />
-        )}
+        {currentAstarteVersion &&
+          isCurrentVersionSupported(currentAstarteVersion, astarteVersionWithPoliciesSupport) &&
+          astarte.token?.can('realmManagement', 'GET', '/policies') && (
+            <Sidebar.Item
+              label="Delivery Policies"
+              link="/trigger-delivery-policies"
+              icon="policy"
+            />
+          )}
```

Comment:
Yes, that's right.
But the Trigger and NewTrigger pages are still accessible, so they should be fixed as well to only fetch the list of delivery policies if they are supported by Realm Management.